src/test/transaction_tests.cpp

```diff
@@ -776,78 +779,4 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     BOOST_CHECK(!IsStandardTx(t, reason));
 }
 
-BOOST_AUTO_TEST_CASE(Test_EnforceZnodePayment)
```

Comment:
Was this test removed by mistake?

Reply:
This test was moved to another file znode_tests. The reason for that is that znode checks are not done on checkTransaction but in checkBlock so another test suite was needed to build actual blockchain and do tests on blocks and not transactions.